src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements AreaCalculator {
```

Comment:
```suggestion
public class IsoscelesTrapezoid extends Figure {
```
the same with other figures, implement this interface in Figure